MessengerOnFileForLinux/DisplayChat/src/NcursesPrintToMainWindowOperationWrapper.cpp

```diff
@@ -0,0 +1,59 @@
+#include "NcursesPrintToMainWindowOperationWrapper.hpp"
+
+#include <ncurses.h>
+#include <chrono>
+#include <thread>
+
+void NcursesPrintToMainWindowOperationWrapper::printInformationIntoMainWindow(const std::string& informationToPrint) const
+{
+    printw(informationToPrint.c_str());
+    refresh();
+    std::this_thread::sleep_for(std::chrono::seconds(2));
+}
+
+void NcursesPrintToMainWindowOperationWrapper::printMainWindow() const
+{
+    printBasesInformationIntoMainWindow({informationToDisplayInMainWindow,
+                                         getenv("USER")});
+
+    printw(">>");
```

Comment:
czemu ten printw nie jest w liście inicjalizacyjnej?

Reply:
bo on był już poza move(2,0)

Reply:
Przekaż go jako string, wtedy masz (std::initializer_list<char *>, std::string) i string wpisuje Ci na końcu, a defaultowy string ustaw pusty to nie będziesz miał z nim problemu. Ewentualnie z jakimiś optionalami można się też pobawić.